front-end/src/pages/main/components/MainCard.jsx

```
@@ -166,37 +166,33 @@ function renderTitle(recentSchedule, scheduleDifferece) {
-          </p>
-          <MediumButton
-            text={'나의 역할 추가하기'}
-            isOutlined={false}
-            onClick={onButtonClick}
-          />
-        </div>;
-  } catch (error) {
-    console.error('카드에러', error);
-  }
+  return (
+      recentSchedule.scheduleMemberNestedDtoList.find(
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)